bangazonreports/views/favorites/customer_fave_sellers.py

```diff
@@ -0,0 +1,56 @@
+import sqlite3
+from django.shortcuts import render
+from bangazonreports.views import Connection
+
+
+def customer_fave_sellers(request):
+    """Function to create an HTML report about which sellers are user favorites"""
+    if request.method == 'GET':
+        with sqlite3.connect(Connection.db_path) as conn:
+            conn.row_factory = sqlite3.Row
+            db_cursor = conn.cursor()
+            select_fave_sellers = """
+                SELECT  user.first_name || ' ' || user.last_name AS seller_name,
+                        fav.seller_id AS seller_id
+                FROM    bangazonapi_favorite AS fav
+                JOIN    bangazonapi_customer AS cust ON cust.id=fav.seller_id
+                JOIN    auth_user AS user ON cust.user_id=user.id
+            """
+            db_cursor.execute("""
```

Comment:
While this works, there is a way to join on a table more than once in the same query if you use a different alias